modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceSalesAgent.testcase

```diff
@@ -663,6 +720,303 @@ definition {
+				CommerceJSONTermsAndConditionsAPI._addCommerceTermsAndConditions(
+					termsCount = 1,
+					termsType = ${termsType});
+			}
+		}
+
+		task ("And 1 Payment Method is enabled(PayPal)") {
+			CommerceChannels.openChannelsAdmin();
+
+			CommerceNavigator.gotoEntry(entryName = "Minium Portal");
+
+			CommerceChannels.editPaymentMethod(
+				active = "true",
+				paymentMethod = "PayPal");
+		}
+
+		task ("And an active discounts applied to the ABS Sensor product is created") {
+			CommerceJSONDiscountsAPI._addDiscount(
+				active = "true",
+				level = "L1",
+				percentageLevel1 = 20,
+				productNames = "ABS Sensor",
+				target = "products",
+				title = "Discount 1",
+				usePercentage = "true");
+		}
+
+		task ("When the Sales Agent navigates to the Account Management page") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "sales.agent@liferay.com");
+
+			Navigator.openWithAppendToBaseURL(urlAppend = "web/minium/account-management");
+		}
+
+		task ("Then can view only the account 1 listed") {
+			AssertElementPresent(
+				key_entryName = "Commerce Account 1",
+				locator1 = "CommerceNavigation#ENTRY_NAME");
+
+			AssertElementNotPresent(
+				key_entryName = "Commerce Account 2",
+				locator1 = "CommerceNavigation#ENTRY_NAME");
+		}
+
+		task ("And can view the Channel Defaults of account 1") {
+			LexiconEntry.gotoEntry(rowEntry = "Commerce Account 1");
+
+			CommerceEntry.gotoMenuTab(menuTab = "Channel Defaults");
+
+			for (var entryNames : list "Billing,Shipping,Delivery,Payment,Price,Discounts,Currencies,Users") {
+				AssertElementPresent(
+					key_termType = ${entryNames},
+					locator1 = "Account#DEFAULT_ACCOUNT_TERMS_AND_CONDITION_PLUS");
+			}
+
+			for (var entryNames : list "ShippingOption,PaymentMethod") {
+				AssertElementPresent(
+					key_firstColumnValue = "Minium Portal",
+					key_secondColumnValue = "Use Priority Settings",
+					key_table = ${entryNames},
+					locator1 = "CommerceEntry#TABLE_ASSERT_TWO_COLUMNS_EDIT_BUTTON");
+			}
+		}
+
+		task ("And can manage all the entries under the Channel Defaults of account 1") {
+			for (var typeList : list "Billing,Shipping") {
+				Account.addDefaultCommerceAddressEntry(
+					addressEntry = "Test Address",
+					addressType = ${typeList},
+					channelEntry = "Minium Portal");
+
+				AssertElementPresent(
+					key_entryDetails = "Test Address",
+					key_termType = ${typeList},
+					locator1 = "Account#DEFAULT_ACCOUNT_TERMS_AND_CONDITION_DROPDOWN");
+			}
+
+			CommerceAccountChannelDefaults.setDefaultPriceList(
+				channelEntry = "Minium Portal",
+				priceListTitle = "Minium Base Price List");
+
+			AssertElementPresent(
+				key_entryDetails = "Minium Base Price List",
+				key_termType = "Price",
+				locator1 = "Account#DEFAULT_ACCOUNT_TERMS_AND_CONDITION_DROPDOWN");
+
+			Account.setDefaultCurrency(currencyName = "Euro");
```

Comment:
After you set as Euro, you make assertion for US Dollar below, it should be Euro I guess.